svelte-kit-scss/src/lib/components/Profile/OrganizationProfile/OrganizationDetails/OrganizationMembers.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { it, expect } from 'vitest';
+import { getByRole, render, screen } from '@testing-library/svelte';
+import OrganizationMembers from './OrganizationMembers.svelte';
+import { MOCK_GITHUB_ORGANIZATION_MEMBERS } from '$lib/helpers/mocks/users';
+
+const mockOrgMembers = MOCK_GITHUB_ORGANIZATION_MEMBERS;
+
+describe('OrganizationMembers', () => {
+  it('should render organization members', () => {
+    render(OrganizationMembers, {
+      members: mockOrgMembers,
+    });
+
+    const links = screen.getAllByRole('link');
+    const images = screen.getAllByRole('img');
+
+    mockOrgMembers.map((member, i) => {
```

Comment:
Although it typically works to create assertions dynamically in tests, it is not a good practice because the system doesn't know what to expect when starting the test. Instead, we could combine `beforeAll` and `it.each` to render the `OrganizationMembers` and have a "separate" test for each member. With each test having the two expects.

A small side note, if you don't care about the results of a `map`, you can use `forEach` instead.